ext/AvizExt/viz/sensitivity.jl

```diff
@@ -175,7 +175,7 @@ Makie figure
 function ADRIA.viz.rsa!(
     g::Union{GridLayout,GridPosition},
     rs::ResultSet,
-    si::NamedDimsArray,
+    si::YAXArrays.Dataset,
```

Comment:
Why not import `Dataset` directly so prefacing it with `YAXArrays` is not necessary?

Or is there a different `Dataset` type that is causing a conflict?

Reply:
I've replaced with `Dataset`